zio/parquetio/reader.go

```diff
@@ -31,7 +32,8 @@ func NewReader(zctx *super.Context, r io.Reader) (*arrowio.Reader, error) {
 		pr.Close()
 		return nil, err
 	}
-	rr, err := fr.GetRecordReader(context.TODO(), nil, nil)
+	cols := columnIndexes(fr.Manifest, fields)
```

Comment:
Shouldn't this check if fields is `nil` and make sure we get all columns?

Reply:
No, if `fields==nil` then `columnIndexes` will return a nil slice of column indexes and that will tell `GetRecordReader` to read all columns.